librz/core/cmd/cmd_open.c

```diff
@@ -836,14 +834,14 @@ static RzCmdStatus open_file(RzCore *core, const char *filepath, ut64 addr, int
 }
 
 RZ_IPI RzCmdStatus rz_open_handler(RzCore *core, int argc, const char **argv) {
-	ut64 addr = argc > 2 ? rz_num_math(core->num, argv[2]) : 0;
+	ut64 addr = argc > 2 ? rz_num_math(core->num, argv[2]) : UT64_MAX;
```

Comment:
why?

Reply:
using 'o <file> <baddr>' does not work in case baddr=0x0

if no baddr is given, use UT64_MAX loader will use advised baddr from file.